sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -345,4 +345,50 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_accelerometer</EFDB_Topic>
+    <Description>LabJack connected accelerometer unit.</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The name of the sensor.</Description>
```

Comment:
Again, what is being "named" here?
A single sensor? Do we expect this to say something like, "AuxTel Truss?"

Reply:
Consider:
The user-defined name of the accelerometer (e.g. AuxTel_accel1).

Reply:
Yes, AuxTel Truss or MTA Azimuth or something like that.

Reply:
Changed.